dist/index.js

```diff
@@ -133,95 +164,148 @@ async function isNotLatestTerraformVersion() {
   return prodVersion != latestVersion;
 }
 
-async function run() {
+function shortSha(fullSha) {
+  return fullSha.slice(0, 7);
+}
+
+function getSha(imageName) {
+  return imageName.split(":").slice(-1)[0];
+}
+
+function getLatestImageUrl(projectName, headSha) {
+  return `${AWS_ECR_URL}/${projectName}:${shortSha(headSha)}`;
+}
+
+async function buildLogs(projects) {
+  let logs = projects.map(async (project) => {
+    const msgsCommits = await getCommitMessages(project.name, project.oldSha);
+    const strCommits = msgsCommits.join("\n");
+    const projectName = project.name.toUpperCase();
+    return `${projectName}\n\n${strCommits}`;
+  });
+  logs = await Promise.all(logs);
+
+  logs = logs.join("\n\n");
+  if (await isNotLatestManifestsVersion()) {
+    logs = `⚠️ **The production version of manifests is behind the latest staging version. Consider upgrading to the latest version before merging this pull request.** \n\n ${logs}`;
+  }
+
+  if (await isNotLatestTerraformVersion()) {
+    logs = `⚠️ **The production version of the Terraform infrastructure is behind the latest staging version. Consider upgrading to the latest version before merging this pull request.** \n\n ${logs}`;
+  }
+  return logs;
+}
+
+async function getContents(owner, repo, path) {
   const { data: data } = await octokit.repos.getContents({
+    owner: owner,
+    repo: repo,
+    path: path,
```

Comment:
```suggestion
    owner,
    repo,
    path,
```

Can be simplified like that I think (variable name as key and then value)

Reply:
👍  
Wasn't sure if modern JS had this (I'm used to TypeScript).